tests.py

```diff
@@ -132,20 +167,21 @@ def fn():
             self.assertEqual(api.client.last_called_data['test_param'], ['42'])
         self.assertRaises(KeyError, fn)
 
-    def test_pairing_status(self):
+    def test_pairing(self):
         api = toopher.ToopherApi('key', 'secret')
         api.client = HttpClientMock({
             'pairings/1': (200,
-                '{"id":"1", "enabled":true, "user":{"id":"1","name":"some user"}}'
+                '{"id":"1", "enabled":true, "pending":true, "user":{"id":"1","name":"some user"}}'
                 )
             })
-        pairing = api.get_pairing_status('1')
+        pairing = api.get_pairing_by_id('1')
         self.assertEqual(api.client.last_called_method, 'GET')
 
         self.assertEqual(pairing.id, '1')
         self.assertEqual(pairing.user_name, 'some user')
         self.assertEqual(pairing.user_id, '1')
         self.assertTrue(pairing.enabled)
+        self.assertTrue(pairing.pending)
```

Comment:
Pairings should not be `enabled` and `pending`. While this isn't a big deal, it could convey the wrong message to any user who reads the tests as documentation.